lib/components/MenuRenderer/testHelper.tsx

```diff
@@ -348,6 +350,80 @@ export const menuTestSuite = ({ name, Component }: MenuTestSuiteParams) => {
+        const { getAllByRole, closeHandler, menuItemHandler } = renderMenu();
+
+        const { menu, menuButton } = getElements({ getAllByRole });
+
+        // Open menu
+        fireEvent.keyUp(menuButton, { keyCode: ENTER });
+        const firstDown = getElements({ getAllByRole });
+        const firstMenuItem = firstDown.menuItems[0];
+
+        // Navigate down one item
+        fireEvent.keyUp(firstMenuItem, { keyCode: ARROW_DOWN });
+        const secondDown = getElements({ getAllByRole });
+        const secondMenuItem = secondDown.menuItems[1];
+
+        // Action the item
+        fireEvent.keyUp(secondMenuItem, { keyCode: SPACE });
+
+        expect(isVisible(menu)).toBe(false);
+        expect(closeHandler).toHaveBeenCalledTimes(1);
+        expect(menuItemHandler).toHaveBeenNthCalledWith(1, 'second');
+        expect(menuButton).toHaveFocus();
+      });
+
+      it('should trigger the click handler on MenuItemLink when selecting it with enter', () => {
+        const { getAllByRole, closeHandler, menuItemHandler } = renderMenu();
+
+        const { menu, menuButton } = getElements({ getAllByRole });
+
+        // Open menu
+        fireEvent.keyUp(menuButton, { keyCode: ENTER });
+        const firstDown = getElements({ getAllByRole });
+        const firstMenuItem = firstDown.menuItems[0];
+
+        // Navigate down two items
+        fireEvent.keyUp(firstMenuItem, { keyCode: ARROW_DOWN });
+        fireEvent.keyUp(firstMenuItem, { keyCode: ARROW_DOWN });
+        const thirdDown = getElements({ getAllByRole });
+        const thirdMenuItem = thirdDown.menuItems[2];
+
+        // Action the item
+        fireEvent.keyUp(thirdMenuItem, { keyCode: ENTER });
+
+        expect(isVisible(menu)).toBe(false);
+        expect(closeHandler).toHaveBeenCalledTimes(1);
+        expect(menuItemHandler).toHaveBeenNthCalledWith(1, 'third');
+        expect(menuButton).toHaveFocus();
+      });
+
+      it('should trigger the click handler on MenuItemLink when selecting it with space', () => {
+        const { getAllByRole, closeHandler, menuItemHandler } = renderMenu();
+
+        const { menu, menuButton } = getElements({ getAllByRole });
+
+        // Open menu
+        fireEvent.keyUp(menuButton, { keyCode: ENTER });
+        const firstDown = getElements({ getAllByRole });
+        const firstMenuItem = firstDown.menuItems[0];
+
+        // Navigate down two items
+        fireEvent.keyUp(firstMenuItem, { keyCode: ARROW_DOWN });
+        fireEvent.keyUp(firstMenuItem, { keyCode: ARROW_DOWN });
+        const thirdDown = getElements({ getAllByRole });
+        const thirdMenuItem = thirdDown.menuItems[2];
+
+        // Action the item
+        fireEvent.keyUp(thirdMenuItem, { keyCode: SPACE });
+
+        expect(isVisible(menu)).toBe(false);
+        expect(closeHandler).toHaveBeenCalledTimes(1);
+        expect(menuItemHandler).toHaveBeenNthCalledWith(1, 'third');
+        expect(menuButton).toHaveFocus();
+      });
```

Comment:
Looks good, can we add a test for selecting the third item (the link) with a mouse click. There is a mouse interactions section up the top.